### PR TITLE
LPS-195466 Reindex log incorrectly states that the execution mode is "null"

### PR DESCRIPTION
**Pull request by felipelorenz**
https://liferay.atlassian.net/browse/LPS-195466

### CODE REVIEW

**Comment by felipelorenz** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexPortalBackgroundTaskExecutor.java [hunk 1]
```diff
@@ -78,10 +81,14 @@ protected void reindex(
 					companyIds);
 
 				if (_log.isInfoEnabled()) {
-					_log.info(
-						StringBundler.concat(
-							"Finished reindexing company ", companyId,
-							" with execution mode ", executionMode));
+					String logMessage =
+						"Finished reindexing company " + companyId;
```
Comment (AlmirFe) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexPortalBackgroundTaskExecutor.java:85]: Do the same in this part of the code

**Review by AlmirFe — CHANGES_REQUESTED** [review]

**Comment by AlmirFe** [comment]
looks good!

**Comment by AlmirFe** [comment]
ci:forward:force

**Pull request closed without merging**